paper_sections/02-gams.Rmd

```diff
@@ -37,39 +43,52 @@ dat$x <- dat$x2
 true <- data.frame(x = sort(dat$x),
                    y = dat$f2[order(dat$x)])
 
-par(mfrow=c(1,3),las=1,mgp=c(2,1,0))
-
-# optimal
+## GCV fit
```

Comment:
should we use `method="REML"` and `method="ML"` throughout?

Reply:
I'll change this